js-runner/src/core/closures.js

```diff
@@ -29,7 +29,7 @@ export function printClosure(c) {
   let closureSize = view[c + 2];
   let closureElts = [];
 
-  // [TODO]: Reenable when this gets ported to AS
+  // TODO: Reenable when this gets ported to AS
```

Comment:
```suggestion
  // TODO: Reenable when this gets ported to Grain
```
Not sure if this is correct. cc @ospencer

Reply:
That whole comment (with the commented out code) can actually be deleted because we haven't supported printing via JS in ages.